odoo/addons/base/models/res_company.py

```diff
@@ -190,8 +190,8 @@ def _company_default_get(self, object=False, field=False):
         _logger.warning("The method '_company_default_get' on res.company is deprecated and shouldn't be used anymore")
         return self.env.company
 
-    # deprecated, use clear_caches() instead
     def cache_restart(self):
+        warnings.warn("Since 16.0, deprecated method, use `clear_caches` instead")
```

Comment:
```suggestion
        warnings.warn("Since 16.0, deprecated method, use `clear_caches` instead", DeprecationWarning, 2)
```

Reply:
Ho god, I always do the same mistake...

Reply:
TBF it's quite annoying that we pretty much only use `warnings.warn` for `DeprecationWarning`, and it's not what that does by default.